app/views/custom_styles/_primer_color_mapping.erb

```diff
@@ -5,15 +5,38 @@
       --body-background: var(--color-canvas-default);
       --body-font-color: var(--color-fg-default);
       --color-accent-fg: var(--content-link-color) !important;
-
-  <% DesignColor.overwritten.each do |design_color| %>
-    <% if design_color.variable == "alternative-color" %>
-        --color-btn-primary-bg:  <%= design_color.hexcode %> !important;
-        --color-btn-primary-hover-bg:  <%= design_color.darken 0.18 %> !important;
-    <% end %>
-    <% if design_color.variable == "content-link-color" %>
-        --color-accent-fg: <%= design_color.hexcode %> !important;
-    <% end %>
-  <% end %>
+  }
+  [data-light-theme=light_high_contrast]{
+    --button--alt-highlight-background-hover-color: var(--color-btn-primary-hover-bg);
+    --button--alt-highlight-background-color: var(--color-btn-primary-bg);
+    --button--alt-highlight-border-color: var(--main-menu-font-color);
```

Comment:
Why did you choose the `main-menu-font-color` for the border of the button? That seems wrong, esecially since you overwrite that variable a few lines later.